v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/v2g_liberty.py

```diff
@@ -333,7 +333,7 @@ async def set_next_action(self, v2g_args=None):
                 )
                 return
 
-            if soc > c.CAR_MIN_SOC_IN_PERCENT and self.in_boost_to_reach_min_soc:
+            if soc >= c.CAR_MIN_SOC_IN_PERCENT and self.in_boost_to_reach_min_soc:
```

Comment:
This looks ok to me now.

Reply:
The next check is on `soc <= (c.CAR_MIN_SOC_IN_PERCENT + 1)`.
Why is the `+ 1` needed?
Wouldn't that mean that it stops loading 1% too soon?

Reply:
Yes, this is to avoid problems with reaching the min-soc, i think the schedule should be requested with this extra "buffer" to avoid the problem.

Reply:
I'll make a new issue to get into the +1 stuff.